index.html

```diff
@@ -126,29 +133,32 @@ <h1 class="project-title">Multi-Post<br>Stories</h1>
-            <div class="intro">
-                <h1>About<br>Myself</h1>
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If
-                    you like what you see and have a project you need coded,
-                    don’t hestiate to contact me.
-                </p>
-            </div>
-                <!-- social medials lins -->
-            <div class="social-medials">
-                <p>LET’S CONNECT</p>
-                <ul class="social-icons">
-                    <li><a href="https://github.com/Ridwanullahi-code"><img src="images/github.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Linkedin.png" alt=""></a></li>
-                    <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
-                    <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
-                    
-                </ul>      
-            </div>
-            <div>
-                <!-- button to download resume -->
-                <a class="button btn" href="">Get my resume</a>
+            <div class="about-summary">
+                <div class="intro">
+                    <h1>About<br>Myself</h1>
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                        you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.
+                    </p>
+                </div>
+                    <!-- social medials lins -->
+                <div class="social-medials">
+                    <p>LET’S CONNECT</p>
+                    <ul class="social-icons">
+                        <li><a href="https://github.com/Ridwanullahi-code"><img src="images/github.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Linkedin.png" alt=""></a></li>
+                        <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
+                        <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
+                        
+                    </ul>      
+                </div>
+                <div>
+                    <!-- button to download resume -->
+                    <a class="button btn" href="">Get my resume</a>
+                </div>
             </div>
+            
             <!-- area-->
             <div id="skills">
```

Comment:
- [ ] kindly update your languages section flex direction for frameworks should be row, and try to edit the overall style of the section to match Figma 

Figma Design            |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/187988729-9d98460a-5356-4247-87eb-a13efd05f214.png)  |  ![image](https://user-images.githubusercontent.com/89282221/187988782-8fa62d3c-b4f1-4e4e-888c-a0c5f7005496.png)